backend/pkg/proto/service.go

```diff
@@ -498,6 +499,18 @@ func (s *Service) protoFileToDescriptor(files map[string]filesystem.File) ([]*de
 		return nil
 	}
 
+	// Add common proto types
```

Comment:
NIT: Can we add a link to a reference here, which tells what other types besides the Google WKT should be included? Just in case we are missing something in the future because something has been added, we can look it up from there ideally.